src/ui.form-radio-group/URadioGroup.vue

```diff
@@ -1,136 +1,23 @@
-    :disabled="disabled"
-    align="topWithDesc"
-    v-bind="groupLabelAttrs"
-    :data-test="dataTest"
-  >
-    <div v-bind="listAttrs">
-      <!-- @slot Use it to add URadio directly. -->
-      <slot>
-        <URadio
-          v-for="(option, index) in options"
-          :key="option.value"
-          :model-value="selectedItem"
-          :value="option.value"
-          :label="option.label"
-          :description="option.description"
-          :disabled="disabled"
-          v-bind="groupRadioAttrs"
-          :data-test="`${dataTest}-item-${index}`"
-        />
-      </slot>
-    </div>
-  </ULabel>
-</template>
-
-<script setup>
+<script lang="ts" setup>
 import { computed, provide } from "vue";
 
 import ULabel from "../ui.form-label/ULabel.vue";
 import URadio from "../ui.form-radio/URadio.vue";
 import { getDefault } from "../utils/ui.ts";
 
-import defaultConfig from "./config.js";
-import { URadioGroup } from "./constants.js";
-import useAttrs from "./useAttrs.js";
-
-defineOptions({ inheritAttrs: false });
-
-const props = defineProps({
-  /**
-   * Radio group selected value.
-   */
-  modelValue: {
-    type: [Boolean, String, Number, Array, Object],
-    default: "",
-  },
-
-  /**
-   * Radio group options.
-   */
-  options: {
-    type: Array,
-    default: () => [],
-  },
-
-  /**
-   * Radio group label.
-   */
-  label: {
-    type: String,
-    default: "",
-  },
-
-  /**
-   * Radio group description.
-   */
-  description: {
-    type: String,
-    default: "",
-  },
-
-  /**
-   * Radio group error message.
-   */
-  error: {
-    type: String,
-    default: "",
-  },
-
-  /**
-   * Radio size.
-   * @values sm, md, lg
-   */
-  size: {
-    type: String,
-    default: getDefault(defaultConfig, URadioGroup).size,
-  },
-
-  /**
-   * Radio group color.
-   * @values brand, grayscale, gray, red, orange, amber, yellow, lime, green, emerald, teal, cyan, sky, blue, indigo, violet, purple, fuchsia, pink, rose
-   */
-  color: {
-    type: String,
-    default: getDefault(defaultConfig, URadioGroup).color,
-  },
-
-  /**
-   * Unique radio group name (sets for each radio).
-   */
-  name: {
-    type: String,
-    required: true,
-  },
+import defaultConfig from "./config.ts";
+import { URadioGroup } from "./constants.ts";
+import useAttrs from "./useAttrs.ts";
 
-  /**
-   * Disable the input.
-   */
-  disabled: {
-    type: Boolean,
-    default: getDefault(defaultConfig, URadioGroup).disabled,
-  },
+import type { URadioGroupProps, SetRadioGroupSelectedItem } from "./types.ts";
 
-  /**
-   * Component config object.
-   */
-  config: {
-    type: Object,
-    default: () => ({}),
-  },
+defineOptions({ inheritAttrs: false });
 
-  /**
-   * Data-test attribute for automated testing.
-   */
-  dataTest: {
-    type: String,
-    default: "",
-  },
+const props = withDefaults(defineProps<URadioGroupProps>(), {
+  size: getDefault<URadioGroupProps>(defaultConfig, URadioGroup).size,
```

Comment:
Return default modelValue. I think it may break component when we don't provide any initial model value.